### DISCUSSION TITLE
LMP 12 support?

### DISCUSSION THREAD

**Discussion by Andrzejone** [nefarius/BthPS3#62] [Support]
![image](https://user-images.githubusercontent.com/39127677/206730671-27e99a41-9044-453a-88f3-cfc85ea825ee.png)
I bought the cheapest BT dongle to run PS3 pads and play Terraria on PC.
Got it, struggled some time to make it run and then checked spec's of it and gave up coz It don't want to run.
I was carefully choosing BT dongle and buy one saying "BT 4.0" [https://www.amazon.pl/gp/product/B0BC1NSK9L/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1](url)
Question is, support is possible? Coz buying another one worst BT dongle seems pain in ass.

**Comment by nefarius** [comment]
The only thing important is the **minimum** required version, everything else is backwards compatible and doesn't need any additional work to support. If it doesn't work for you maybe you shouldn't have settled for the cheapest crap out there 😉

**Comment by Andrzejone** [comment]
If I could, yes.

**Reply by nefarius** [reply]
Well, that's nothing we can fix in software 😇

**Comment by Andrzejone** [comment]
> Well, that's nothing we can fix in software 😇


actually I mamanged to fix this.
just cleaned everything couple of times and finnaly it worked.
Just leaving the feedback here
